### ISSUE TITLE
Option --help creates an empty diagnostics file

### ISSUE THREAD

**Issue by robin13** [elastic/support-diagnostics#209]
When running `diagnostics.sh` with the `--help` option, a file is generated:

```
rclarke@rcl-laptop:support-diagnostics-6.4.2$ ./diagnostics.sh --help
Using /usr/lib/jvm/java-8-oracle/bin/java as Java Runtime
Using -Xms256m -Xmx2000m  for options.
14:09:44 INFO  Validating inputs...
...
14:09:44 INFO  Writing diagnostic manifest.
14:09:44 INFO  Configuring REST endpoint.
14:09:44 INFO  Archiving diagnostic results.
rclarke@rcl-laptop:support-diagnostics-6.4.2$ ls -al
total 52
drwxr-xr-x   3 rclarke rclarke  4096 Jul 17 14:09 .
drwxrwxr-x 120 rclarke rclarke 20480 Jul 17 08:58 ..
-rw-r--r--   1 rclarke rclarke    10 Jul 17 14:09 -20180717-140944.tar.gz    <---- This is newly created
-rw-r--r--   1 rclarke rclarke   579 Jun  1 17:39 diagnostics.bat
-rwxr-xr-x   1 rclarke rclarke   682 Jun  1 17:39 diagnostics.sh
drwxr-xr-x   2 rclarke rclarke  4096 Jul 13 08:36 lib
-rw-r--r--   1 rclarke rclarke   575 Jun 22 11:08 scrub.bat
-rwxr-xr-x   1 rclarke rclarke   677 Jun 22 11:08 scrub.sh
-rw-r--r--   1 rclarke rclarke    78 Jun 22 11:08 scrub.yml
rclarke@rcl-laptop:support-diagnostics-6.4.2$ 
```

**Comment by nemonster** [comment]
Will be fixed in the next release.

**Comment by nemonster** [comment]
Addressed in 6.4.4.
